int64 jointIndex
string jointName
int64 jointType
string jointTypeStr
int64 qIndex
int64 uIndex
int64 flags
float64 jointDamping
float64 jointFriction
float64 jointLowerLimit
float64 jointUpperLimit
float64 jointMaxForce
float64 jointMaxVelocity
string linkName
geometry_msgs/Point jointAxis
geometry_msgs/Point parentFramePos
geometry_msgs/Quaternion parentFrameOrn
int64 parentIndex